src/pykiso/logging_initializer.py

```diff
@@ -234,3 +239,94 @@ def initialize_loggers(loggers: Optional[List[str]]) -> None:
     loggers_to_deactivate = set(relevant_loggers) - set(active_loggers)
     for logger_name in loggers_to_deactivate:
         logging.getLogger(logger_name).setLevel(logging.WARNING)
+
+
+def import_object(path: str) -> Any:
+    """return the object based on the path.
+        For example : logging.Logger will return Logger
+
+    :param path: path to the object
+    """
+    if path:
+        components = path.split(".")
+        mod = importlib.import_module(components[0])
+        for comp in components[1:]:
+            mod = getattr(mod, comp)
+        return mod
+    else:
+        return None
+
+
+def add_filter_to_handler(func):
+    """Decorator function that will add a filter to all the handlers
+        of a logger after a function.
+
+    :param func: function to execute
+    """
+
+    def wrapper(self, *arg, **kwargs):
+        func(self, *arg, **kwargs)
+        for handler in self.handlers:
+            handler.addFilter(InternalLogsFilter())
+
+    return wrapper
+
+
+def remove_handler_from_logger(func):
+    """Decorator that will remove all handlers of a logger after
+        executing a function.
+
+    :param func: function to execute
+    """
+
+    def wrapper(self, *arg, **kwargs):
+        func(self, *arg, **kwargs)
+        for handler in self.handlers:
+            self.removeHandler(handler)
+
+    return wrapper
+
+
+def change_logger_class(log_level: str, verbose: bool, logger: str):
+    """Change the class of all the logger of pykiso.
```

Comment:
It also change the sublevels of the logger right ? Because you call it only once..